data_diff/dbt_parser.py

```diff
@@ -49,7 +40,10 @@ def legacy_profiles_dir() -> Path:
 
 class DbtParser:
     def __init__(self, profiles_dir_override: str, project_dir_override: str) -> None:
-        self.parse_run_results, self.parse_manifest, self.ProfileRenderer, self.yaml = import_dbt()
+        self.parse_run_results = parse_run_results
+        self.parse_manifest = parse_manifest
+        self.ProfileRenderer = ProfileRenderer
+        self.yaml = yaml
```

Comment:
Maybe we should remove this `self.whatever` approach here and use the functions/classes directly as they are imported (as we do everywhere all the time normally) — to reduce the unnecessary code complexity and contributor confusion.

Reply:
good point, updated